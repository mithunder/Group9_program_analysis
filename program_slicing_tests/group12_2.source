module group122 :
x:= 1;
y:=0;
z:=0;
w:= 0;
do  (x <  20)  -> {
@programslicing="true"
  z:= x / 3;
  w:=w+w;
  y:= z * 3;
  if ( z = y) ->
    write x
  fi;
  x:= x+1
}
od
end
