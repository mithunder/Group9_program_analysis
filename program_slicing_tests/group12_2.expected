module group122 :
x:= 1;
do  (x <  20)  -> {
  z:= x / 3;
  x:= x+1
}
od
end
